packages/postman-to-openapi/src/convert.ts

```diff
@@ -0,0 +1,771 @@
+  return name != null ? { license: { name, ...(url ? { url } : {}) } } : {}
+}
+
+function parseContact(
+  variables: PostmanVariable[],
+  optsContact: any = {},
+): any {
+  const nameVar = getVarValue(variables, 'contact.name')
+  const urlVar = getVarValue(variables, 'contact.url')
+  const emailVar = getVarValue(variables, 'contact.email')
+  const { name = nameVar, url = urlVar, email = emailVar } = optsContact
+  return [name, url, email].some((e) => e != null)
+    ? {
+        contact: {
+          ...(name ? { name } : {}),
+          ...(url ? { url } : {}),
+          ...(email ? { email } : {}),
+        },
+      }
+    : {}
+}
+
+function parseExternalDocs(
+  variables: PostmanVariable[],
+  optsExternalDocs: any,
+): any {
+  const descriptionVar = getVarValue(variables, 'externalDocs.description')
+  const urlVar = getVarValue(variables, 'externalDocs.url')
+  const { description = descriptionVar, url = urlVar } = optsExternalDocs
+  return url != null
+    ? { externalDocs: { url, ...(description ? { description } : {}) } }
+    : {}
+}
+
+function parseBody(body: any = {}, method: string): any {
+  if (['GET', 'DELETE'].includes(method)) return {}
+  const { mode, raw, options = { raw } } = body
+  let content: any = {}
+  switch (mode) {
+    case 'raw': {
+      const { raw: rawOptions = {} } = options
+      const { language } = rawOptions
+      let example: any = ''
+      if (language === 'json') {
+        if (raw) {
+          const errors: any[] = []
+          example = parse(raw, errors)
+          if (errors.length > 0) {
+            example = raw
+          }
+        }
+        content = {
+          'application/json': {
+            schema: {
+              type: 'object',
+              example,
+            },
+          },
+        }
+      } else if (language === 'text') {
+        content = {
+          'text/plain': {
+            schema: {
+              type: 'string',
+              example: raw,
+            },
+          },
+        }
+      } else {
+        content = {
+          '*/*': {
+            schema: {
+              type: 'string',
+              example: JSON.stringify(raw),
+            },
+          },
+        }
+      }
+      break
+    }
+    case 'file':
+      content = {
+        'text/plain': {},
+      }
+      break
+    case 'formdata': {
+      content = {
+        'multipart/form-data': parseFormData(body.formdata),
+      }
+      break
+    }
+    case 'urlencoded':
+      content = {
+        'application/x-www-form-urlencoded': parseFormData(body.urlencoded),
+      }
+      break
+  }
+  return { requestBody: { content } }
+}
+
+function parseFormData(data: any[]): any {
+  const objectSchema: any = {
+    schema: {
+      type: 'object',
+    },
+  }
+  return data.reduce((obj: any, { key, type, description, value }: any) => {
+    const { schema } = obj
+    if (isRequired(description)) {
+      ;(schema.required = schema.required || []).push(key)
```

Comment:
This is hard to parse syntax. This can be broken into two lines or done differently.